odoo/addons/base/models/res_lang.py

```diff
@@ -47,6 +47,18 @@ class Lang(models.Model):
              "Provided ',' as the thousand separator in each case.")
     decimal_point = fields.Char(string='Decimal Separator', required=True, default='.', trim=False)
     thousands_sep = fields.Char(string='Thousands Separator', default=',', trim=False)
+    position = fields.Selection([
+        ('after', 'After Amount'),
+        ('before', 'Before Amount')], default='after', required=True,
+        string='Currency Symbol Position', help="Determines where the currency symbol should be placed after or before the amount.")
+    sign_position = fields.Selection([
+        ('0', 'Surrounded by parentheses'),
+        ('1', 'Before value and symbol'),
+        ('2', 'After value and symbol'),
+        ('3', 'Before value'),
+        ('4', 'After value')], string='Sign Position',
+        default='1', help="Determines where the currency symbol should be placed after or before the amount.")
+    is_space = fields.Boolean(string='Allow space between amount and currency symbol')
```

Comment:
This is a field on the language, calling it `is_space` is not very clear as implying the language is a space? :-/